client/src/main/kotlin/no/nav/su/se/bakover/client/stubs/person/PersonOppslagStub.kt

```diff
@@ -12,38 +12,43 @@ import no.nav.su.se.bakover.domain.person.PersonOppslag.KunneIkkeHentePerson
 
 object PersonOppslagStub :
     PersonOppslag {
-    override fun person(fnr: Fnr): Either<KunneIkkeHentePerson, Person> = Person(
-        ident = Ident(fnr, AktørId("2437280977705")),
-        navn = Person.Navn(
-            fornavn = "Tore",
-            mellomnavn = "Johnas",
-            etternavn = "Strømøy"
-        ),
-        telefonnummer = Telefonnummer(landskode = "+47", nummer = "12345678"),
-        adresse = listOf(
-            Person.Adresse(
-                adresselinje = "Oslogata 12",
-                bruksenhet = "U1H20",
-                poststed = Person.Poststed(postnummer = "0050", poststed = "OSLO"),
-                kommune = Person.Kommune(kommunenummer = "0301", kommunenavn = "OSLO"),
-                adressetype = "Bostedsadresse",
-                adresseformat = "Vegadresse"
-            )
-        ),
-        statsborgerskap = "NOR",
-        kjønn = "MANN",
-        adressebeskyttelse = null,
-        skjermet = false,
-        kontaktinfo = Person.Kontaktinfo(
-            epostadresse = "mail@epost.com",
-            mobiltelefonnummer = "90909090",
-            reservert = false,
-            kanVarsles = true,
-            språk = "nb"
-        ),
-        vergemål = null,
-        fullmakt = null
-    ).right()
+    override fun person(fnr: Fnr): Either<KunneIkkeHentePerson, Person> =
+        if (fnr.toString() == "15034435674") {
```

Comment:
Dette kan kanskje være et ekte fødselsnummer? Isåfall bør vi nok håndtere det litt annerledes.

Reply:
F.eks noe `startsWith` eller noe sånt.

Reply:
Godt poeng. Det ble hentet fra en tilfeldig fnr-generator, men teoretisk sett så kan det være at det treffer en ekte person ja. Jeg tar og gjør om sånn at man eventuelt kan legge inn lokalt i `.env` i stedet.